test/mero_SUITE.erl

```diff
@@ -332,16 +332,20 @@ multiget_defineds(_Conf) ->
     ?assertMatch({ok, 3}, mero:increment_counter(cluster, <<"16">>)),
     %% 13, 14 and 15 will go to the same server
     %% 11, 12 and 16 to a different one
-    Expected = [{<<"17">>, undefined},
-                {<<"15">>, <<"2">>},
-                {<<"14">>, <<"2">>},
-                {<<"13">>, <<"1">>},
-                {<<"16">>, <<"3">>},
-                {<<"12">>, <<"1">>},
-                {<<"11">>, <<"1">>}],
+    Expected = lists:sort([{<<"17">>, undefined},
```

Comment:
You could've just sorted the list in place, instead of adding a `lists:sort/1` call ;)